compiler/onecc-docker/onecc-docker

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+''''exec "python3" "$0" "$@" #'''
+
+# Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import subprocess
+import json
+import os
+import requests
+
+
+def _is_image(version) : 
```

Comment:
```suggestion
def _image_exists(name) : # or _image_exists(tag) 
```

1. `_is_image` sounds like it checks if the thing is image or not. How about use `exist` verb?
2. Receiving `name` directly rather than version seems more intuitive and would reduce the duplication of information.

Reply:
> The previous plan seems to cause you a lot of confusion.
Would it be better to exclude those parts?

Actually, I think whether the previous plan exists or not doesn't matter. Even if the plan existed, the function name still would have seemed to check if given version of image exists or not.

I don't know what you wrote in the first place where you used to consider various versions but even considering multiple versions, the explanation in the above my comment would not be changed. Just a method listing the release versions might be needed later.

In short, you don't have to exclude those parts because we should always consider extensibility. I'm reviewing the codes with that mind:) 

Please let me know if you have any comments!